Doc/library/ast.rst

```diff
@@ -1644,15 +1663,93 @@ Pattern matching
                                     value=Constant(value=Ellipsis))])])],
             type_ignores=[])
 
+.. _ast-type-params:
+
+Type parameters
+^^^^^^^^^^^^^^^
+
+:ref:`Type parameters <type-params>` can exist on classes, functions, and type
+aliases.
+
+.. class:: TypeVar(name, bound)
+
+   A :class:`~typing.TypeVar`. ``name`` is the name of the type variable, and
```

Comment:
I'd prefer it if we used the fully qualified name here (and elsewhere in this document), or the fact that there's both a `typing.TypeVar` and an `ast.TypeVar` could cause confusion